src/consumer/component/consumer-component.js

```diff
@@ -521,6 +530,12 @@ export default class Component {
     return JSON.stringify(this.toObject());
   }
 
+  calculateMainDistFile(): string {
```

Comment:
Please add a TODO comment here which says:
This need to be calculated by the result of the compiling process (in bit map or in the model) because it might be that the compiler will change the mainFile name (for example replace main.ts by main.js)